randovania/data/json_data/prime3.txt

```diff
@@ -2067,59 +2427,86 @@ Hall of Remembrance
 Asset id: 6915612318130423578
```

Comment:
Add node "Event - Hall of Remembrance Item" with connections to "Dock to Imperial Crypt (Lower)" and "Dock to Tower (Lower)" with requirement Screw Attack